core/src/layers/prometheus.rs

```diff
@@ -88,12 +88,62 @@ use crate::*;
 #[derive(Default, Debug, Clone)]
 pub struct PrometheusLayer {
     registry: Registry,
+    requests_duration_seconds_buckets: Vec<f64>,
+    bytes_total_buckets: Vec<f64>,
 }
 
 impl PrometheusLayer {
     /// create PrometheusLayer by incoming registry.
+    /// For promtheus layer, we will use default buckets for HistogramMetric.
+    /// Be careful, if you want to use custom buckets, please use [`PrometheusLayer::with_registry_and_custom_metric_bucket`].
+    /// Or set buckets directly by [`PrometheusLayer::requests_duration_seconds_buckets`] and [`PrometheusLayer::bytes_total_buckets`] if you need it.
     pub fn with_registry(registry: Registry) -> Self {
-        Self { registry }
+        let mut result = Self {
+            registry,
+            ..Default::default()
+        };
+        result.requests_duration_seconds_buckets = exponential_buckets(0.01, 2.0, 16).unwrap();
+        result.bytes_total_buckets = exponential_buckets(0.01, 2.0, 16).unwrap();
+        result
+    }
+
+    /// create PrometheusLayer by incoming registry and custom buckets.
+    pub fn with_registry_and_custom_metric_bucket(
```

Comment:
I perfer to remove this API.

- This API is overlap with `requests_duration_seconds_buckets` and `bytes_total_buckets` which doesn't make sense.
- Adding new options in the future will introduce a breaking change.